### PR TITLE
docs: fix imports of cheerio in examples

### PR DESCRIPTION
**Pull request by B4nan**
Related: https://github.com/apify/apify-docs/discussions/661

### CODE REVIEW

**Comment by B4nan** [comment]
cc @metalwarrior665, I've left only one occurence of the named import (`import {load} from 'cheerio'`) as it was described in that specific section, so it felt file, but we might wanna change that too.

https://docs.apify.com/academy/puppeteer-playwright/executing-scripts/collecting-data#parsing-in-node-context

File: sources/academy/tutorials/node_js/dealing_with_dynamic_pages.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ So, we've gotta scroll down the page to load these images. Luckily, because we'r
 
 ```js
 import { PuppeteerCrawler, utils, Dataset } from 'crawlee';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
```
Comment (barjin) [inline sources/academy/tutorials/node_js/dealing_with_dynamic_pages.md:147]: Can't we just get rid of the cheerio import here whatsoever? It's unused(?), and might confuse beginners that to use the `parseWithCheerio` helper, you need to import the cheerio library.

File: sources/academy/webscraping/puppeteer_playwright/common_use_cases/paginating_through_results.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ Let's grab this number now with a little bit of code:
 
 ```javascript
 import { chromium } from 'playwright';
-import { load } from 'cheerio';
+import * as cheerio from 'cheerio';
```
Comment (barjin) [inline sources/academy/webscraping/puppeteer_playwright/common_use_cases/paginating_through_results.md:56]: Same here, maybe I'm missing something though (or maybe it's that we've used cheerio in this example before?)

**Comment by barjin** [comment]
There is a bunch of other places where the Cheerio imports are unused(?), I would vouch for removing those altogether

**Review by barjin — CHANGES_REQUESTED** [review]

File: sources/academy/tutorials/node_js/dealing_with_dynamic_pages.md (hunk 1 above)
Comment (B4nan) [reply sources/academy/tutorials/node_js/dealing_with_dynamic_pages.md:147]: right, saw it in one place but forgot to check others, good catch

File: sources/academy/webscraping/puppeteer_playwright/common_use_cases/paginating_through_results.md (hunk 2 above)
Comment (barjin) [inline sources/academy/webscraping/puppeteer_playwright/common_use_cases/paginating_through_results.md:56]:
```suggestion
```

File: sources/academy/webscraping/puppeteer_playwright/common_use_cases/paginating_through_results.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ await browser.close();
 
 ```javascript
 import puppeteer from 'puppeteer';
-import { load } from 'cheerio';
+import * as cheerio from 'cheerio';
```
Comment (barjin) [inline sources/academy/webscraping/puppeteer_playwright/common_use_cases/paginating_through_results.md:82]:
```suggestion
```

**Review by barjin — APPROVED** [review]

**Pull request merged**